Refit/Buffers/PooledBufferWriter.Stream.cs

```diff
@@ -91,9 +91,12 @@ public override Task CopyToAsync(Stream destination, int bufferSize, Cancellatio
 
                 try
                 {
+#if NETSTANDARD2_1
```

Comment:
Does this also need to be `|| NET5_0`?

Reply:
Thank you, yes. Not for the library to work, but for the tests to actually test this code path.